libs/mrwebrtc/src/data_channel.cpp

```diff
@@ -74,21 +74,29 @@ bool DataChannel::Send(const void* data, size_t size) noexcept {
   if (data_channel_->buffered_amount() + size > GetMaxBufferingSize()) {
     return false;
   }
+
+  auto state = data_channel_->state();
+  RTC_LOG_F(LS_INFO) << state;
```

Comment:
Remove? Or add contextual message and not just the state itself?

Reply:
This is a debug leftover, will remove.

Reply:
Fixed